torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2777,6 +2808,26 @@ def reference_lgamma(x):
                         # see discussion https://github.com/pytorch/pytorch/pull/47761#issuecomment-747316775
                         SkipInfo('TestGradients', 'test_fn_gradgrad', device_type='cuda'),)
                     ),
+    OpInfo('linalg.det',
+           op=torch.linalg.det,
+           aliases=('det', ),
+           dtypes=floating_and_complex_types(),
+           aten_name='linalg_det',
+           sample_inputs_func=sample_inputs_linalg_det,
+           decorators=[skipCUDAIfNoMagma, skipCPUIfNoLapack],
+           supports_complex_autograd=False,
+           supports_inplace_autograd=False,
+           skips=(
+               # The following tests fail only on ROCm. This is probably
+               # related to the fact that the current linalg.det backward is
+               # unstable if the matrix has repeated singular values, see
+               # https://github.com/pytorch/pytorch/issues/53364
+               SkipInfo('TestGradients', 'test_fn_grad', device_type='cuda',
+                        dtypes=[torch.float64], active_if=TEST_WITH_ROCM),
+               SkipInfo('TestCommon', 'test_variant_consistency_jit', device_type='cuda',
+                        dtypes=[torch.float64, torch.float32], active_if=TEST_WITH_ROCM),
```

Comment:
Nit prefer tuples to lists if the iterable doesn't need to be mutated

Reply:
d962b375f4